### PR TITLE
fix(apcd-cms): fp-1873 register form, footnotes

### PR DESCRIPTION
**Pull request by wesleyboar**
## Overview / Changes

- Fix order of footnotes.
- Use markup for footnotes, not unicode.﹡

<sup>﹡ Unicode footnote character sizes are rendered inconsistently with some fonts.</sup>

## Related

- [FP-1873](https://jira.tacc.utexas.edu/browse/FP-1873)

## Testing

1. Log in to CMS.
2. Load form at http://localhost:8000/register/request-to-submit/.
3. Confirm footnotes appear in form in sequential order.
4. Confirm footnotes appear beneath form in matching order.

## UI

<details><summary>Tall Screenshot <sub>with arrows pointing to changed text</sub></summary>

fp-1884-fp-1873-form-render

</details>

### CODE REVIEW

**Review by wesleyboar — COMMENTED** [review]
I updated this branch with the latest (submission form and stuff) from `epic/...` branch.

But still, it does **not yet** update the dynamic fields. I had done that in #23, but there are merge conflicts now.

I will try again tonight.

**Comment by wesleyboar** [comment]
Dynamic markup footnotes updated (e2ed9f3). Ready for review.

File: apcd-cms/src/apps/registrations/templates/submission_form/submission_form.html [hunk 1]
```diff
@@ -1990,12 +1990,12 @@ <h6>
...
         >¹ Third Party Administrator / Administrative Services Only (TPA/ASO)<br />
-        ² Federal Employer Identification Number (FEIN)<br />
-        ³ National Association of Insurance Commissioners (NAIC)<br />
+        ² Secure File Transfer Protocol (SFTP)<br />
+        ³ Hypertext Transfer Protocol Secure (HTTPS)<br />
         ⁴ United States Dollar&nbsp;(USD)<br />
-        ⁵ Secure File Transfer Protocol (SFTP)<br />
-        ⁶ Hypertext Transfer Protocol Secure (HTTPS)<br />
-        ⁷ Universal Serial Bus (USB)</small
+        ⁵ Universal Serial Bus (USB)<br />
```
Comment (Strmiska) [inline apcd-cms/src/apps/registrations/templates/submission_form/submission_form.html:1996]: I think you meant for USB to be 4 and USD to be 5

File: apcd-cms/src/apps/registrations/templates/submission_form/submission_form.html [hunk 2]
```diff
@@ -1990,12 +1990,12 @@ <h6>
...
+        ⁶ National Association of Insurance Commissioners (NAIC)<br />
+        ⁷ Federal Employer Identification Number (FEIN)</small
```
Comment (Strmiska) [inline apcd-cms/src/apps/registrations/templates/submission_form/submission_form.html:1998]: Some for FEIN, I think the reference numbers got mixed up a bit.

File: apcd-cms/src/apps/registrations/templates/submission_form/submission_form.html (hunk 1 above)
Comment (wesleyboar) [reply apcd-cms/src/apps/registrations/templates/submission_form/submission_form.html]: Thank you. 8244fda

**Review by Strmiska — APPROVED** [review]
LGTM!

**Pull request merged**